Dockerfile

```diff
@@ -1,10 +1,15 @@
 # Builder image
-FROM golang:1.10.1-alpine as builder
+FROM golang:1.11.2-alpine as builder
 
 RUN apk add --no-cache \
+    ca-certificates \
```

Comment:
Is this necessary?

Reply:
You are right, this isn't needed for TiDB